CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Master (Unreleased)
 
+## 2.23.0 (2023-05-28)
+- Update the message output of `RSpec/ExpectActual` to include the word 'value'. ([@corydiamand])
```

Comment:
Please move this above 2.23

Reply:
Thanks for the help @pirj! For this, would you like me to move line 6 under ## Master (Unreleased) and leave the space under ## 2.23.0 (2023-05-28) blank?